src/server/game/Entities/Player/PlayerTaxi.cpp

```diff
@@ -61,16 +62,21 @@ void PlayerTaxi::InitTaxiNodesForLevel(uint32 race, uint32 chrClass, uint8 level
         SetTaximaskNode(213);                               //Shattered Sun Staging Area
 }
 
-void PlayerTaxi::LoadTaxiMask(std::string const& data)
+bool PlayerTaxi::LoadTaxiMask(std::string const& data)
```

Comment:
Dont fail. 0 on invalid tokens

Reply:
Same as faction change, this has the potential for irreversible data loss once the character gets saved back to DB.